### PR TITLE
ember-intl upgrade

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: tests/integration/components/project-settings/view-scenario/details-column-header-test.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, triggerEvent } from '@ember/test-helpers';
+import { find, render, triggerEvent } from '@ember/test-helpers';
```
Comment (future-pirate-king) [inline tests/integration/components/project-settings/view-scenario/details-column-header-test.js:3]: is find being used ?

File: tests/test-utils.js [hunk 2]
```diff
@@ -33,3 +34,24 @@ export function objectifyEncodedReqBody(reqBody = '') {
...
+  assert,
+  { element, selector, message, doIncludesCheck }
+) => {
+  const innerHTML =
+    element?.innerHTML || find(selector.trim()).innerHTML.trim();
+
+  if (doIncludesCheck) {
+    assert.true(
+      innerHTML.includes(message),
+      `Element HTML: "${innerHTML}" <=====> Compared HTML: "${message}"`
```
Comment (future-pirate-king) [inline tests/test-utils.js:48]: assert message can be defined to variable and re-used

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/account-settings/general/select-language/index.ts [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { task } from 'ember-concurrency';
-import IntlService from 'ember-intl/services/intl';
 import { action } from '@ember/object';
+import { tracked } from 'tracked-built-ins';
```
[the remark this answers is not in the document]
Comment (Yibaebi) [reply app/components/account-settings/general/select-language/index.ts:5]: fixed

File: tests/integration/components/file/report-drawer/va-reports/index-test.js [hunk 4]
```diff
@@ -288,7 +288,7 @@ module(
 
     test('it renders the previous, summary, and current reports if at least two reports have been generated', async function (assert) {
       // Creates two file-reports
-      this.server.createList('file-report', 2, {
+      const reportList = this.server.createList('file-report', 2, {
```
[the remark this answers is not in the document]
Comment (Yibaebi) [reply tests/integration/components/file/report-drawer/va-reports/index-test.js:291]: resolved

File: tests/integration/components/project-settings/view-scenario/details-column-header-test.js (hunk 1 above)
Comment (Yibaebi) [reply tests/integration/components/project-settings/view-scenario/details-column-header-test.js:3]: resolved

File: tests/test-utils.js (hunk 2 above)
Comment (Yibaebi) [reply tests/test-utils.js:48]: Updated

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**